.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+* @remyleone
```

Comment:
Il manque inference et edge_services.

Reply:
They are not yet present on the main branch.